Chart.yaml

```diff
@@ -53,8 +53,8 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 0.2.2
+version: 0.2.3
```

Comment:
Shouldn't we sync this version with `appVersion`?

Reply:
I don't think these have to match. But yes, it should be bumped according to the comment above.